scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -92,6 +92,10 @@ SG_ State : 0|8@1+ (1,0) [0|255] "" LOGGER
 
 BO_ 306 FSM_AIR_SHUTDOWN: 0 FSM
 
+BO_ 307 FSM_FLOW_METER: 8 FSM
+SG_ Primary_flow_meter_flow_rate : 0|32@1+ (1,0) [0|0] "" LOGGER 
+SG_ Secondary_flow_meter_flow_rate : 32|32@1+ (1,0) [0|0] "" LOGGER
```

Comment:
```suggestion
SG_ Primary_Flow_Rate : 0|32@1+ (1,0) [0|0] "L/min" LOGGER 
SG_ Secondary_Flow_Rate : 32|32@1+ (1,0) [0|0] "L/min" LOGGER
```

nit units, also flow meter is in the message name so no need to mention it again in signal name